src/web/components/SharingPermission/SharingPermissionsTable.stories.tsx

```diff
@@ -1,14 +1,40 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react';
 
+import { AvailableParticipantDTO } from '../../../api/routers/participantsRouter';
+import { TestAvailableSiteListProvider } from '../../services/site';
 import { SharingPermissionsTable } from './SharingPermissionsTable';
 
 export default {
   title: 'Sharing Permissions/SharingPermissionTable',
   component: SharingPermissionsTable,
 } as ComponentMeta<typeof SharingPermissionsTable>;
 
+const response: AvailableParticipantDTO[] = [
+  {
+    name: 'Site 1',
+    types: [{ id: 1, typeName: 'Type 1' }],
+    siteId: 10,
+  },
+  {
+    name: 'Site 2',
+    types: [{ id: 2, typeName: 'Type 2' }],
+    siteId: 11,
+  },
+  {
+    name: 'Site 3',
+    types: [{ id: 3, typeName: 'Type 3' }],
+    siteId: 12,
+  },
+  {
+    name: 'Site 4',
+    types: [{ id: 4, typeName: 'Type 4' }],
+    siteId: 13,
+  },
+];
 const Template: ComponentStory<typeof SharingPermissionsTable> = (args) => (
-  <SharingPermissionsTable {...args} />
+  <TestAvailableSiteListProvider response={response}>
```

Comment:
Given we can provide test sites for this storybook using this Provider, I wonder if it is worth removing the storybook `ParticipantSearchBar.stories.tsx` since it is essentially the same thing, but prop driven. I was confused as to what this line of code did because I was accidentally looking at `Sharing Permissions/ParticipantSearchBar` which kept working after commenting out this line 😅 What do you think?

Reply:
I assume you mean `ParticipantSearchBar.stories` and `SearchAndAddParticipants.stories` are the same thing? Happy to remove one of them, but as we have the tests on  ParticipantSearchBar, how about removing `SearchAndAddParticipants.stories` instead?

Reply:
hmm actually I was thinking of removing `ParticipantSearchBar.stories` but you are right in that there are tests that depend on it. Let's keep both for now since they are ever so slightly different, we can always revisit later if it is still confusing